python/lsst/ip/isr/assembleCcdTask.py

```diff
@@ -195,13 +197,15 @@ def assembleCcd(self, assembleInput):
         ccd = None
         if hasattr(assembleInput, "has_key"):
             # Get a detector object for this set of amps
-            ccd = assembleInput.itervalues().next().getDetector()
+            ccd = assembleInput.values().next().getDetector()
```

Comment:
I'm not sure it matters: we just want to grab one detector from the input stuff, so the fact that it's a generator should be fine.